src/dosbox.cpp

```diff
@@ -615,29 +615,36 @@ void DOSBOX_Init(void) {
 #endif
 #if C_FLUIDSYNTH
 		"fluidsynth",
+#endif
+#if C_MT32EMU
+		"mt32",
 #endif
 		"none",
-		0
-	};
+		0 };
+
 	pstring->Set_values(midi_devices);
 	pstring->Set_help(
 	        "Device that will receive the MIDI data (from the emulated MIDI\n"
-	        "interface - MPU-401).\n"
+	        "interface - MPU-401). Choose one of the following:\n"
 #if C_FLUIDSYNTH
-	        "Use 'fluidsynth' to select built-in software synthesiser,\n"
-	        "see the fluidsynth section for detailed configuration.\n"
+	        "'fluidsynth', to use the built-in MIDI synthesiser. See the\n"
+	        "       [fluidsynth] section for detailed configuration.\n"
+#endif
+#if C_MT32EMU
+	        "'mt32', to use the built-in Roland MT-32 synthesiser.\n"
+	        "       See the [mt32] section for detailed configuration.\n"
 #endif
-	        "Use 'auto' to pick the first working external device.");
+	        "'auto', to use the first working external MIDI player. This\n"
+	        "       might be a software synthesizer or physical device.");
 
 	pstring = secprop->Add_string("midiconfig", when_idle, "");
 	pstring->Set_help(
 	        "Configuration options for the selected MIDI interface.\n"
 	        "This is usually the id or name of the MIDI synthesiser you want\n"
 	        "to use (find the id/name with DOS command 'mixer /listmidi').\n"
-#if C_FLUIDSYNTH
-	        "- When using the built-in FluidSynth (mididevice = fluidsynth),\n"
-	        "  this option has no effect. Instead, configure FluidSynth using\n"
-	        "  the dedicated [fluidsynth] section below.\n"
+#if (C_FLUIDSYNTH == 1 || C_MT32EMU == 1)
+	        "- This option has no effect when using the built-in sythensizers\n"
```

Comment:
Typo: `sythensizers`

Reply:
Spelling emergency here 🚑 . 
@FeignInt , we need a `./scripts/verify-spelling.sh`, STAT 👨‍⚕️ 😅

Reply:
I just pushed updates to #763 ( after rebase which I neglected to do earlier )

It works, however..

I piped this through it and sadly :cry: it didn't picked up sythensizers

The problem with spellintian is that it relies on "common misspelt" word list

I'll see if I can find something that checks with more vigour

Reply:
Relying on common misspellings is good - in other project I tried to run all code comments via hunspell and it was pointing out all technical and programming-related terms as spelling errors - it was very annoying to work with in the long run...

Reply:
yeah, I imagine this is why spellintian just uses the word list it has

what I have done is make the script quite passive, in that it will only fail a job if the typo is in the binary

I can expand this a little and have a more "fool proof" spellcheck just Annotate with the lowest "debug" level

Reply:
Indeed.. rampant false-positives are far more damning:  pilots learn to ignore constant warnings which has lead to crashed planes in prior generations.. so great effort today goes to preventing "alarm fatigue".  The medical industry still has rampant alarm fatigue (as anyone who's spent time in ER will attest to) and it's still killing people, unfortunately. 

Luckily the only killing I'm doing with bad spelling is to the English language :smile: But we might as we follow best practices.  So I like you're existing happy-medium @FeignInt.

Reply:
Fixed in eddf117c